util/cron/test-llvm.bash

```diff
@@ -1,12 +1,23 @@
 #!/usr/bin/env bash
 #
-# Test default configuration with CHPL_LLVM=llvm and pass --llvm flag to
-# compiler on linux64.
+# Full test suite with CHPL_LLVM=llvm and pass --llvm flag to
+# compiler on linux64. Now uses paratest.server.
+
+# Needs /data/cf/chapel/setup_python27.bash (common-llvm)
 
 CWD=$(cd $(dirname $0) ; pwd)
 source $CWD/common.bash
 source $CWD/common-llvm.bash
+source $CWD/common-localnode-paratest.bash
+
+# common-llvm restricts us to extern/ferguson, but we want all the tests
+unset CHPL_NIGHTLY_TEST_DIRS
+
+# paratest with 8 nodes on localhost
+nightly_args="${nightly_args} $(set +x ; get_nightly_paratest_args 8)"
```

Comment:
Let's keep an eye on this.  It should work, but if we start getting sporadic failures, we may need to dial this down to 4 nodes instead of 8.